src/test/java/net/whitbeck/rdbparser/RdbParserTest.java

```diff
@@ -832,6 +841,57 @@ public void hashmapAsZipList() throws Exception {
     }
   }
 
+  @Test
+  public void hashmapAsListpack() throws Exception {
+    if (rdbVersion > 9) {
```

Comment:
Minor, but the rest of this file uses `>= X` to indicate "introduce in version X or later".

```suggestion
    if (rdbVersion >= 10) {
```